.github/workflows/update_dependencies.yml

```diff
@@ -0,0 +1,37 @@
+name: Update dependencies
+
+on:
+    workflow_dispatch:
+    schedule:
+        # at 00:00 on Monday (UTC)
+        - cron: "0 0 * * MON"
```

Comment:
❓: How about adding this to the existing `chores.yml`? If this needs to run on Monday, update that file so that both jobs run on Monday instead of Sunday.

Reply:
I initially thought that I would add it to `chores.yml` but I'm thinking it's going to be pointless to have this run every week when we really only need it based on the first week of a sprint. Since that schedule would be difficult to generate, what if we only had this use `workflow_dispatch` so that who ever wants to update dependencies can just come in and click a button? Otherwise, we will be in a perpetual game of "close the stale dependency update pull requests".

Reply:
Works for me.